Mathlib/CategoryTheory/Filtered/Basic.lean

```diff
@@ -963,4 +964,22 @@ instance [IsCofiltered C] [IsCofiltered D] : IsCofiltered (C × D) where
 
 end Prod
 
+section Connected
+
+instance [IsFilteredOrEmpty C] : IsPreconnected C :=
```

Comment:
I think these already exist in `Filtered/Connected.lean`.

Reply:
Any reason you didn't make them instances there?

Reply:
Presumably because they would always apply and I didn't want to fiddle with priorities and think about whether I'm introducing a cycle.